.buildkite/dagster-buildkite/dagster_buildkite/steps/dagster.py

```diff
@@ -70,6 +71,20 @@ def build_repo_wide_ruff_steps() -> List[CommandStep]:
     ]
 
 
+def build_repo_wide_prettier_steps() -> List[CommandStep]:
+    return [
+        CommandStepBuilder(":prettier: prettier")
+        .run(
+            "pushd js_modules/dagster-ui/packages/eslint-config",
+            "yarn install",
+            "popd",
+            "make check_prettier",
+        )
+        .on_test_image(AvailablePythonVersion.get_default())
```

Comment:
will follow up with a skip rule

Reply:
follow up in this PR or another? "theres a changed file ending in yml/yaml" should be easy enough i would hope

Reply:
done